qutebrowser/browser/webkit/network/filescheme.py

```diff
@@ -42,9 +42,9 @@ def get_file_list(basedir, all_files, filterfunc):
     """
     items = []
     for filename in all_files:
-        absname = os.path.join(basedir, filename)
+        absname = pathlib.Path(basedir / filename)
```

Comment:
if basedir/filename are strings this seems like it won't work. If they are paths, it seems like you don't need the `pathlib.Path` here

Reply:
(I'm guessing you meant `pathlib.Path(basedir) / filename`?)